ccmlib/scylla_docker_cluster.py

```diff
@@ -17,6 +17,8 @@ class ScyllaDockerCluster(ScyllaCluster):
     def __init__(self, *args, **kwargs):
         super(ScyllaDockerCluster, self).__init__(*args, **kwargs)
         self.docker_image = kwargs['docker_image']
+        self.ipprefix = '172.17.0.'
+        self.ipformat = '{}%d'.format(self.ipprefix)
```

Comment:
Why not use with f-string?

Reply:
I agree, them better. just save legacy pattern and forgot to change.